packages/swap/src/event-handlers/swapExecuted.ts

```diff
@@ -54,17 +54,25 @@ export default async function swapExecuted({
     throw new Error(`swapExecuted: No existing swap entity for swap ${swapId}.`);
   }
 
+  let swapInputAmount = BigInt(swap.swapInputAmount.toFixed());
+
   const fees = await calculateIncludedSwapFees(
     swap.srcAsset,
     swap.destAsset,
-    BigInt(swap.swapInputAmount.toFixed()),
+    swapInputAmount,
     intermediateAmount,
     swapOutput,
   );
 
+  if (swap.srcAsset === 'Usdc') {
+    const networkFee = fees.find((type) => type.type === 'NETWORK')!;
+    swapInputAmount -= BigInt(networkFee.amount);
```

Comment:
do we need to remember to change this again after 1.5?

Reply:
Yeah but I have a ticket already

Reply:
added a comment